frontend/src/components/Product/ProductTitle/ProductTitle.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Text, theme } from '@fun-eat/design-system';
+import { useState } from 'react';
+import styled from 'styled-components';
+
+import { SvgIcon } from '@/components/Common';
+import productDetail from '@/mocks/data/productDetail.json';
+
+const ProductTitle = () => {
+  const { name, bookmark } = productDetail;
+  const [isBookmarked, setIsBookmarked] = useState(bookmark);
+
+  const toggleBookmark = () => {
+    setIsBookmarked(!isBookmarked);
+  };
+
+  return (
+    <ProductTitleContainer>
+      <ProductTitleWrapper>
+        <Button color="white" variant="filled">
+          <SvgIcon variant="arrow" color={theme.colors.gray5} width={15} height={15} />
+        </Button>
+        <Text
+          as="span"
+          css={`
+            margin-left: 10px;
+          `}
```

Comment:
```suggestion
          css="margin-left: 10px;"
```

안에 변수를 사용하는 것이 없으니 중괄호 제거해도 됩니당